mern-ecommerce/controllers/auth.js

```diff
@@ -73,10 +73,12 @@ exports.isAuth = (req, res, next) => {
 };
 
 exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
-    return res.status(403).json({
-      error: 'Admin resource! Access denied',
+  if (req.profile.role == 0) { 
```

Comment:
Here is the review:

In the line `if (req.profile.role == 0) {`, consider using strict comparison `===` instead of `==` to ensure both value and type are checked, preventing unexpected behavior due to type coercion.